src/components/ChatWindow/MessageList/TextMessage/TextMessage.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import clsx from 'clsx';
+import { makeStyles, createStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    messageContainer: {
+      borderRadius: '16px',
+      display: 'inline-flex',
+      alignItems: 'center',
+      padding: '0.5em 0.8em 0.6em',
+      margin: '0.3em 0 0',
+      wordBreak: 'break-word',
+      backgroundColor: '#E1E3EA',
+      hyphens: 'auto',
+    },
+    isLocalParticipant: {
+      backgroundColor: '#CCE4FF',
+    },
+  })
+);
```

Comment:
When the `theme` object isn't needed, we can just pass a plain object to `makeStyles`:
```suggestion
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles({
  messageContainer: {
    borderRadius: '16px',
    display: 'inline-flex',
    alignItems: 'center',
    padding: '0.5em 0.8em 0.6em',
    margin: '0.3em 0 0',
    wordBreak: 'break-word',
    backgroundColor: '#E1E3EA',
    hyphens: 'auto',
  },
  isLocalParticipant: {
    backgroundColor: '#CCE4FF',
  },
});
```